src/components/views/elements/MemberEventListSummary.js

```diff
@@ -107,10 +107,17 @@ module.exports = React.createClass({
                 </span>
             );
         }
+
+        // Return null instead of an empty <span> if there is no joinSummary
```

Comment:
ok, but why would we be here in that case? no joinsummary and no leavesummary implies that both `joiners` and `leavers` are null, which implies that `joinEvents` and `leaveEvents` are both empty? so, why would that happen?